src/analytics.js

```diff
@@ -62,16 +69,13 @@ window.addEventListener( 'scroll', () => {
 } );
 
 /**
- * Campaign.
+ * Query string parameters.
  */
-const params = new URLSearchParams( window.location.search );
-const utm = {
-	utm_source: params.get( 'utm_source' ) || '',
-	utm_medium: params.get( 'utm_medium' ) || '',
-	utm_campaign: params.get( 'utm_campaign' ) || '',
-	utm_term: params.get( 'utm_term' ) || '',
-	utm_content: params.get( 'utm_content' ) || '',
-};
+const params = parseQueryString( window.location.search );
+const qv_params = {};
```

Comment:
nitpick: `qvParams`?

Reply:
Yep, don't know I had a blank on that one!